sympy/printing/mathml.py

```diff
@@ -13,34 +13,89 @@
 from .conventions import split_super_sub, requires_partial
 
 
-class MathMLPrinter(Printer):
-    """Prints an expression to the MathML markup language
+class MathPrinter(Printer):
+    # Contains common code required for MathMLContentPrinter and
+    # MathMLPresentationPrinter
 
-    Whenever possible tries to use Content markup and not Presentation markup.
-
-    References: https://www.w3.org/TR/MathML3/
-    """
     printmethod = "_mathml"
```

Comment:
Define different `printmethod` for each printer.

Reply:
I am thinking of renaming printmethods as `printmethod = "_mathml_content" ` and ` printmethod = "_mathml_presentation" ` . Will it be fine ?